aries-site/src/pages/foundation/color.js

```diff
@@ -94,7 +94,7 @@ const Color = () => (
           some of the ways we illustrate our brand.
```

Comment:
Can we update the "See in Figma" button to point to: https://www.figma.com/file/eZYR3dtWdb9U90QvJ7p3T9/hpe-design-system-library-color instead of where it currently is? (Line 79)

Reply:
yes thanks for pointing that out!